### PR TITLE
Remove cereal

### PR DESCRIPTION
**Pull request by IrinaShkviro**
Remove cereal from the project

### CODE REVIEW

File: third_party/conan/lockfiles/linux/clang7_debug/conan.lock [hunk 1]
```diff
@@ -574,174 +570,174 @@
...
     "options": "fPIC=True\nminizip=False\nshared=False",
     "build_requires": [
-     "90"
+     "88"
     ]
    },
-   "84": {
-    "pref": "openssl/1.0.2t#a108e98ed2578863c45746ff2bd2512d:fc1522b9e4333a8c74ca8b19c266b1939a89f941",
+   "82": {
+    "pref": "openssl/1.0.2t#146f912909a48f790ee66ca0cc3716b6:fc1522b9e4333a8c74ca8b19c266b1939a89f941",
```
Comment (beckerhe) [inline third_party/conan/lockfiles/linux/clang7_debug/conan.lock:710]: This openssl change shouldn't be here. Can you revert the change of the recipe revision? Doing that manually is probably the simplest solution.

Comment (beckerhe) [reply third_party/conan/lockfiles/linux/clang7_debug/conan.lock:710]: Unfortunately this has to be done twice per file for all lockfiles.

File: third_party/conan/lockfiles/windows/msvc2017_debug/conan.lock [hunk 2]
```diff
@@ -602,811 +598,803 @@
...
    },
-   "127": {
+   "125": {
     "pref": "cmake_installer/3.16.3@conan/stable#0:60b04a80ac4dd5c23f1acffacc679e9902e636e5",
     "options": ""
    },
-   "128": {
-    "pref": "jom/1.1.3#dc792c80fc49a53c5e043caf9c60d385:3475bd55b91ae904ac96fde0f106a136ab951a5e",
+   "126": {
+    "pref": "jom/1.1.3#200b42aeebff3e448d17f58138608474:3475bd55b91ae904ac96fde0f106a136ab951a5e",
```
Comment (beckerhe) [inline third_party/conan/lockfiles/windows/msvc2017_debug/conan.lock:976]: This recipe revision change shouldn't be here.

File: third_party/conan/lockfiles/linux/clang7_debug/conan.lock (hunk 1 above)
Comment (IrinaShkviro) [reply third_party/conan/lockfiles/linux/clang7_debug/conan.lock]: Fixed, thanks for noticing!

File: third_party/conan/lockfiles/windows/msvc2017_debug/conan.lock (hunk 2 above)
Comment (IrinaShkviro) [reply third_party/conan/lockfiles/windows/msvc2017_debug/conan.lock]: Fixed
